front/src/layouts/WithBackground.vue

```diff
@@ -15,17 +15,18 @@
       class="p-col"
       style="max-width: 600px"
     >
-      <router-link
-        class="p-d-block p-text-center"
-        to="/"
-      >
-        <img
-          alt="logo"
-          :src="logoBgPrimaryURL"
-          class="p-mb-4 p-mx-auto"
-        />
-      </router-link>
-
+      <div v-if="logoBgPrimaryURL">
+        <router-link
+          class="p-d-block p-text-center"
+          to="/"
+        >
+          <img
+            alt="logo"
+            class="p-mb-4 p-mx-auto"
+            :src="logoBgPrimaryURL"
+          />
+        </router-link>
+      </div>
```

Comment:
I think you could directly put the `v-if` directive in the `router-link` element ?

But maybe you ever try it ?

```suggestion
      <router-link
        v-if="logoBgPrimaryURL"
        class="p-d-block p-text-center"
        to="/"
      >
        <img
          alt="logo"
          class="p-mb-4 p-mx-auto"
          :src="logoBgPrimaryURL"
        />
      </router-link>
```

Reply:
you are right